Projects/TEAM-1/Turbal Y/Selenium_1/Selenium_1/Selenium_1/Program.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Selenium_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
```

Comment:
We no need an empty Main method. We should just change the Output type of the project to ClassLibrary

<img width="808" alt="image" src="https://user-images.githubusercontent.com/15199521/169763879-0099ff93-e0f8-41c1-b821-26f7deb19518.png">